backend/windmill-api/src/flows.rs

```diff
@@ -85,6 +85,38 @@ pub fn global_service() -> Router {
         .route("/hub/get/:id", get(get_hub_flow_by_id))
 }
 
+fn validate_flow_value(flow_value: &serde_json::Value) -> Result<()> {
+    #[cfg(not(feature = "enterprise"))]
+    if flow_value
+        .get("ws_error_handler_muted")
+        .map(|val| val.as_bool().unwrap_or(false))
+        .is_some_and(|val| val)
+    {
+        return Err(Error::BadRequest(
+            "Muting the error handler for certain flow is only available in enterprise version"
+                .to_string(),
+        ));
+    }
+
+    if let Some(modules) = flow_value.get("modules").and_then(|m| m.as_array()) {
```

Comment:
it only go through first level, there is likely a dfs function somewhere however I think it might be the wrong approach, you likely want to try to deserialize each module and fail at first module that can't be deserializing with giving the id + error from the deserializer. Much more generic. You can then just apply a validate function on a FlowModuleValue directly for anything post validation.